ci/logging_tests_controller.yml

```diff
@@ -23,6 +25,25 @@
       - openshift
       - openstack-operators
       - openshift-logging
+    common_endpoint_test_id: "RHOSO-12682"
+    common_endpoint_list:
+      - [nova,compute,public]
+      - [nova,compute,internal]
+      - [placement,placement,public]
+      - [placement,placement,internal]
+      - [swift,object-store,public]
+      - [swift,object-store,internal]
+      - [cinderv3,volumev3,public]
+      - [cinderv3,volumev3,internal]
+      - [barbican,key-manager,public]
+      - [barbican,key-manager,internal]
+      - [keystone,identity,public]
+      - [keystone,identity,internal]
+      - [glance,image,public]
+      - [glance,image,internal]
+      - [neutron,network,public]
+      - [neutron,network,internal]
```

Comment:
Technically, we only need the 0th and second items in this list, but there's some new changes in progress in https://github.com/infrawatch/feature-verification-tests/pull/203 that will address this